receiver/vcenterreceiver/scraper.go

```diff
@@ -269,6 +269,9 @@ func (v *vcenterMetricScraper) collectVMs(
 			continue
 		}
 
+		if moVM.Config == nil {
+			continue
```

Comment:
Is this an unexpected condition? Should we add an error to `errs`?

Reply:
At present, I am not sure what caused this situation. My guess is that the virtual machine was disconnected during the collection process, but such a problem should not cause the process to stop.
What kind of error message should I add?

Reply:
I think just something like `errs.AddPartial(1, fmt.Errorf("vm config empty for %s", <some id if possible>))